compiler/damlc/daml-stdlib-src/DA/Internal/Prelude.daml

```diff
@@ -445,6 +445,24 @@ span p xs@(x::xs')
          | p x          =  let (ys,zs) = span p xs' in (x::ys,zs)
          | otherwise    =  ([],xs)
 
+-- | The `partition` function takes a predicate, a list and returns
+-- the pair of lists of elements which do and do not satisfy the
+-- predicate, respectively; i.e.,
+--
+-- > partition p xs == (filter p xs, filter (not . p) xs)
+--
+-- ```
+-- >>> partition (`elem` "aeiou") "Hello World!"
```

Comment:
Oh this doesn’t actually work (use `bazel test //compiler/damlc:daml-stdlib-doctest` to reproduce locally) becaues string literals are not lists of characters in DAML. Maybe just turn it into something like `partition (> 5) [1,7,3,8]`.